medical/models/medical_patient.py

```diff
@@ -107,3 +108,17 @@ def _get_default_image_path(self, vals):
         return get_module_resource(
             'medical', 'static/src/img', 'patient-avatar.png',
         )
+
+    def _search_age(self, operator, value):
+        current_date = date.today()
+        current_year = current_date.year
+        current_day = current_date.day
+        first_possible_birthdate = current_date.replace(
+            year=current_year - (int(value) + 1)
+        )
+        last_possible_birthdate = first_possible_birthdate.replace(
+            year=current_year - int(value),
+            day=current_day - 1
+        )
+        return ['&', ('birthdate_date', '>=', first_possible_birthdate),
```

Comment:
This only supports the `=` operator. We also need logic to support `>`, `>=`, `<`, `<=`